qownnotes/snapcraft.yaml

```diff
@@ -10,7 +10,7 @@ description: |
 
 apps:
   qownnotes:
-    command: desktop-launch usr/bin/QOwnNotes --snap
+    command: desktop-launch ${SNAP}/usr/bin/QOwnNotes --snap
```

Comment:
We try to simplify things and give best practices in the playpen. As snapcraft is prepending your internal version first, I think it makes sense to just reference it? (We could argue the same about "if there is a desktop-launch" software on your system.

In any case, here, those internal versions will be what's used, so let's keep the short and simple versions, makes sense?